cvr/src/test/java/org/cbioportal/cmo/pipelines/cvr/CVRClinicalRecordTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+/*
+ * This file is part of cBioPortal CMO-Pipelines.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package org.cbioportal.cmo.pipelines.cvr;
+
+import org.cbioportal.cmo.pipelines.cvr.model.staging.CVRClinicalRecord;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+/**
+ *
+ * @author ochoaa
+ */
+@ContextConfiguration(classes=CvrTestConfiguration.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class CVRClinicalRecordTest {
+
+    @Test
+    public void testSampleTypeNull() throws Exception {
+        CVRClinicalRecord record = new CVRClinicalRecord();
+        String sampleType = record.resolveSampleType(null);
+        Assert.assertTrue(sampleType.isEmpty());
+    }
+
+    @Test
+    public void testSampleTypeNotNull() throws Exception {
+        CVRClinicalRecord record = new CVRClinicalRecord();
+        String primarySampleType = record.resolveSampleType(0);
+        String metastasisSampleType = record.resolveSampleType(1);
+        String localRecurrenceSampleType = record.resolveSampleType(127);
+        String defaultSampleType = record.resolveSampleType(128);

```

Comment:
according to the card, sample type 127 should resolve to the unknown sample type. Other codes should maybe resolve to null / empty / or cause an exception?  I'm not sure what is appropriate.